packages/admin-stories/src/apollo-swapi-story.decorator.tsx

```diff
@@ -0,0 +1,44 @@
+import { ApolloClient, ApolloLink, ApolloProvider, createHttpLink, InMemoryCache } from "@apollo/client";
+import { onError } from "@apollo/client/link/error";
+import { ErrorScope, errorScopeForOperationContext, useErrorDialog } from "@comet/admin";
+import { StoryContext, StoryFn } from "@storybook/addons/dist/types";
+import * as React from "react";
+
+const SwapiApolloProvider: React.FunctionComponent = ({ children }) => {
+    const errorDialog = useErrorDialog();
+    const link = ApolloLink.from([
+        onError(({ graphQLErrors, networkError, operation }) => {
```

Comment:
can't we have a ApolloProvider in comet-admin that implements this? (now you have to copy this into each application, right?)

Reply:
what exactly do you mean with 

> can't we have a ApolloProvider in comet-admin that implements this? 

Do you think we should wrap the Apollo Provider inside of @comet/admin?

I tend more in a direction that we make a custom Apollo Link - `ErrorDialogLink` that supports this kind of Error Dialog intersection Logic automatically and use this in our projects

Reply:
> I tend more in a direction that we make a custom Apollo Link

of course that's the only way to do that :)

Reply:
added a custom `createErrorDialogApolloLink` Method